src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
```

Comment:
The class `ColorSupplier` should not have an empty line at the beginning of its definition. Please remove the empty line to adhere to Java best practices.